https-upgrades/tests.json

```diff
@@ -42,7 +42,8 @@
                 "requestType": "main_frame",
                 "expectURL": "http://negativetest.com/path/file.html?search=query&another=one#fragment",
                 "exceptPlatforms": [
-                    "android-browser"
+                    "android-browser",
+                    "ios-browser"
```

Comment:
we have a separate tag for macos ("macos-browser") - https://github.com/duckduckgo/privacy-reference-tests/blob/main/CONTRIBUTING.md#exceptplatforms . Are BSK tests checking only "ios-browser" tag? Or are they checking for both? Or are those tests actually in iOS?

Reply:
I'm aware of that, these tests are implemented inside the BSK, so they test the common code for both platforms. Ideally we should have apple-browsers or apple-platforms, but Bartek told me that for now we should just check against ios-browser inside the BSK (I believe we do it in couple of tests already). However, I'm open to any suggestions here.

Reply:
all good, just wanted to understand how you are using those tags 👍